### ISSUE TITLE
Header image

### ISSUE THREAD

**Issue by filibert0** [SnowdaysIT/snowdays-front-end#22]
On the header image, "Snowdays19" watermark is visible on in the bottom-left corner at most widescreen resolutions, or at least I see it on my 1920px windows laptop with chrome.

Consider avoiding the background linear gradient in light blue on the image (` .clear-filter[filter-color=blue]`): for me, it ruins completely the image, it doesn't add anything to the website, and it does not relate in any way to snowdays' graphic and photographic styles. 
Consider also bringing back the "panorama" animation magic there was in the old website, it was great, and would give a feeling of continuity to users that remember the old website.

**Comment by FraTheVet** [comment]
Aight imma put it in a bullet list:
1. "Snowdays19" watermark on header image
2. Avoiding background linear gradient
3. "panorama" animation magic

PR #12 takes care of 1. (new image format) and partially 3. (fixed header + first section animation and image positioning)

WIth regards to 2. I actually tried that but it seems that it conflicts in readability for some viewports. I actually tried mirroring it in the first section (as the image itself is mirrored) but it looked awful. Will still try out some other options (that PR is a draft) and keep you posted

Thanks for your feedback :)
